config/3.3/event-manager/templates/noi-diffing-customization-sa.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Shall we move this to `argocd/openshift-gitops`?

Reply:
This diff ingnore is necessary when installing eventmanager separately (not through all in one).

Reply:
I'm suspecting argocd ConfigMap does not work in OpenShift GitOps. It will be reverted by operator. Right now, w/ the contents merged from sandbox, all Argo customization will be held in config/argocd. Shall we move it there?

Reply:
I have removed this configmap. It does not take effect. This serviceaccount will be auto synced during the sync operation, so it is no longer necessary to apply for diff ignore.